### ISSUE TITLE
Fluid property 

### ISSUE THREAD

**Issue by FabricioCVitorino** [kevindorma/psvpy#1]
Hi! First of the all, outstanding contribution. My question is does the code consider fluid properties and phase changing? Sorry if this is a trivial question but I'm undergraduating student still learning. Thanks!

**Comment by kevindorma** [comment]
No, single phase only. Use Homogeneous Equilibrium Method for phase change. I will post a new set of engineering tools in a few weeks. Thanks. --Kevin Dorma, Ph.D., P.Eng. (AB,MB)Process Engineering ConsultantPresident, Kevin Dorma Consulting, Ltd.E: ***@***.***: (403) 700-9801LinkedIn: https://www.linkedin.com/in/kevin-dorma-69a60232On Oct 22, 2024, at 7:49 PM, FabricioCVitorino ***@***.***> wrote:﻿
Hi! First of the all, outstanding contribution. My question is does the code consider fluid properties and phase changing? Sorry if this is a trivial question but I'm undergraduating student still learning. Thanks!

**Comment by FabricioCVitorino** [comment]
All right. Thank you Mr. @kevindorma I'll be tuned in to check your updates.
Cheers
